tests/info/test_h5.py

```diff
@@ -17,6 +19,38 @@ def test_output_matrices(centered_pair_predictions):
     assert len(names) == 27
     assert isinstance(names[0], np.string_)
 
+    # Check that node names and edges are read correctly
+    node_names = [
+        n.decode() for n in get_nodes_as_np_strings(centered_pair_predictions)
+    ]
+    edge_names = [
+        (s.decode(), d.decode())
+        for (s, d) in get_edges_as_np_strings(centered_pair_predictions)
+    ]
+
+    assert node_names[0] == "head"
+    assert node_names[1] == "neck"
+    assert node_names[2] == "thorax"
+    assert node_names[3] == "abdomen"
+    assert node_names[4] == "wingL"
+    assert node_names[5] == "wingR"
+    assert node_names[6] == "forelegL1"
+    assert node_names[7] == "forelegL2"
+    assert node_names[8] == "forelegL3"
+    assert node_names[9] == "forelegR1"
+    assert node_names[10] == "forelegR2"
+    assert node_names[11] == "forelegR3"
+    assert node_names[12] == "midlegL1"
+    assert node_names[13] == "midlegL2"
+    assert node_names[14] == "midlegL3"
+    assert node_names[15] == "midlegR1"
+    assert node_names[16] == "midlegR2"
+    assert node_names[17] == "midlegR3"
+
+    for (src_node, dst_node) in edge_names:
+        assert src_node in node_names
+        assert dst_node in node_names
```

Comment:
Check for correctness here instead.

```py
assert edge_names == centered_pair_predictions.skeleton.edge_names
```

if sequence comparison with lists doesn't work, try:

```py
assert tuple(edge_names) == tuple(centered_pair_predictions.skeleton.edge_names)
```

Reply:
Sequence comprehension works!